### PR TITLE
tests for get_user fucntion

### LINKED ISSUE #47: #18: Write tests for get_user fucntion
1) Read guideline [here](https://gist.github.com/iamnovichek/801ac395a06a7d8b04b022084ace684d)
2) Write tests for `get_user` from `tracker/utils.py` in `tracker/tests/test_utils.py`
3) Provide some screenshots to prove that your tests work

### PR DESCRIPTION
**Pull request by ikemHood**
Closes #47 

- tested for valid user and invalid user

### CODE REVIEW

File: tracker/tests/test_utils.py [hunk 1]
```diff
@@ -1,15 +1,18 @@
+import os
 import django
 
-django.setup()
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'core.settings')
```
Comment (iamnovichek) [inline tracker/tests/test_utils.py]: Remove this line pls.

File: tracker/tests/test_utils.py [hunk 2]
```diff
@@ -1,15 +1,18 @@
...
 
 from asgiref.sync import async_to_sync
 from django.test import TestCase
 from faker import Faker
 
 from tracker.choices import Roles
 from tracker.models import CustomUser, Repository, TelegramUser
-from tracker.utils import get_all_repostitories
+from tracker.utils import get_all_repostitories, get_user
+from uuid import UUID
```
Comment (iamnovichek) [inline tracker/tests/test_utils.py]: Use `isort your_file`

**Review by iamnovichek — CHANGES_REQUESTED** [review]
Provide some screenshots that your tests work

**Comment by ikemHood** [comment]
Screenshot from 2024-11-30 17-54-42

**Pull request merged**